script.js

```diff
@@ -1,4 +1,233 @@
+// Create a basic version of Scissors Paper Stone where
+// the user inputs one of "scissors", "paper", or "stone",
+// the program internally randomly chooses scissors, paper, or stone,
+// and the program outputs whether the user won, the program won, or it's a draw.
+
+// global variables
+var gameMode = "waiting for username";
+var username = "";
+var playCount = 0;
+var winCount = 0;
+var mjpWin = "no one"; // korean SPS winner
+
+// SPS function randomises the computer choice
+function SPS() {
+  var randomiser = Math.floor(Math.random() * 3); // using randomiser to pick a number
+  var computer = ["scissors", "paper", "stone"];
+  return computer[randomiser]; // returning the results of the randomised choice
+}
```

Comment:
Wow, pretty advanced but good that you used an array to help select the outcomes haha